### ISSUE TITLE
a question about bninception backbone

### ISSUE THREAD

**Issue by Dyfine** [kakaoenterprise/AAAI2021_MDR#2]
Hi, thanks for the great repo of your work MDR, it's really an inspiring perspective. Currently I have looked through the code and find in `AAAI2021_MDR/model/backbone/bninception.py` that all the relu operations in the bninception are not in use. I feel puzzled about this and is there any reason about this setting？

**Comment by gkyh0805** [comment]
Thanks for letting me know.

We used those files from other repos (originally from https://github.com/Cadene/pretrained-models.pytorch).

I checked those files in other repos and the original repo.

Most of repos do not use relu, and the original repo modified that two years ago.

The omission of relu is not intended, and, by correcting them, the performance can be slightly increased.
(but the original repo reported that the difference of performance is negligible.)
![image](https://user-images.githubusercontent.com/34022221/117952243-f0487200-b34f-11eb-978c-bc65143d323e.png)
